models/issue_list.go

```
@@ -146,13 +148,16 @@ func (issues IssueList) loadLabels(e Engine) error {
 			var labelIssue LabelIssue
 			err = rows.Scan(&labelIssue)
 			if err != nil {
-				rows.Close()
+				errclose := rows.Close()
+				if errclose != nil {
+					log.Error("IssueList.loadLabels: %v", errclose)
+				}
 				return err
 			}
 			issueLabels[labelIssue.IssueLabel.IssueID] = append(issueLabels[labelIssue.IssueLabel.IssueID], labelIssue.Label)
 		}
-		rows.Close()
-		left = left - limit
+		_ = rows.Close()
```

Why is there different handling of error on `Close()`?
L159, L248, L301, L453: error is ignored
L345, L393: error is returned to caller

---

Mhh good point. I've changed it so the error is always returned.